docs/developer/tips.md

```diff
@@ -0,0 +1,12 @@
+# Developer tips and best practices
+
+## Using Providers for MyST content
+
+When using components that render MyST content (using `<MyST ast={content} />`), they must be wrapped wrap them in various providers e.g. `TabStateProvider` to ensure proper React context is available.
+These providers facilitate an alternative to prop drilling, the name used to describe passing props down through several React components. They are also used to isolate state between different subtrees, e.g. `TabStateProvider` defines a synchronised tab context.
```

Comment:
```suggestion
These providers facilitate an alternative to prop drilling, the name used to describe passing props down several layers of React components.
```

>  They are also used to isolate state between different subtrees, e.g. `TabStateProvider` defines a synchronised tab context.

Maybe that should stay, but I'm not sure user has ability to understand without further explanation.